tool/tbot/config/configtemplate_ssh_client.go

```diff
@@ -134,6 +135,10 @@ func (c *TemplateSSHClient) Describe() []FileDescription {
 	}
 }
 
+// sshConfigUnsupportedWarning is used to ensure we don't spam log messages if
+// using non-filesystem backends.
+var sshConfigUnsupportedWarning sync.Once
+
 func (c *TemplateSSHClient) Render(ctx context.Context, authClient auth.ClientI, currentIdentity *identity.Identity, destination *DestinationConfig) error {
```

Comment:
It would be super nice to include a test for this function using the `Golden` helper, so we can easily see what the rendered file looks like in diffs

Reply:
I've gone ahead and done this in https://github.com/gravitational/teleport/pull/12898 . It should be simple enough to pull that commit into this branch, or approve and merge that PR and then rebase onto it.

You'll need to run `GOLDEN_UPDATE=1 go test -timeout 10m -run ^TestTemplateSSHClient_Render$ github.com/gravitational/teleport/tool/tbot/config` to regenerate the golden file for your changes in this PR.

Reply:
I am a big fan of making the result of these changes obvious during review. Any objections if we try to get this in after v9.3?

Reply:
Ah, yep, I've fixed the failing test, making `ssh_config` optional broke it.

I do like the golden approach but let's aim to include that post-v9.3.